src/store/index.ts

```diff
@@ -95,4 +95,9 @@ export const focusTaskTitle = atom({
   default: "",
 });
 
+export const flipAnimationState = atom({
```

Comment:
why we are making it a global state ? and not use it as local state?

Reply:
if we use local state, the header component re-renders when handlePartner() is called which resets the flipping state to false